packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -21,17 +22,20 @@ const packageManager = getPackageManager();
 describe('Create new sdk command', () => {
   setupLocalRegistry();
   beforeEach(async () => {
-    sdkFolderPath = await prepareTestEnv(appName, 'blank');
-    sdkPackagePath = path.join(sdkFolderPath, sdkPackageName.replace(/^@/, ''));
+    const isYarnTest = packageManager.startsWith('yarn');
+    const yarnVersion = isYarnTest ? getO3rYarnVersion(process.cwd()) || 'latest' : undefined;
+    sdkFolderPath = await prepareTestEnv(appName, 'blank', yarnVersion);
+    sdkPackagePath = path.posix.join(sdkFolderPath, sdkPackageName.replace(/^@/, ''));
     execAppOptions.cwd = sdkFolderPath;
 
-    if (packageManager.startsWith('yarn')) {
-      fs.writeFileSync(path.join(sdkFolderPath, 'package.json'), '{"name": "@test/sdk"}');
+    if (isYarnTest) {
+      fs.writeFileSync(path.posix.join(sdkFolderPath, 'package.json'), `{"name": "@test/sdk", "packageManager": "yarn@${yarnVersion}"}`);
+      packageManagerInstall(execAppOptions);
 
       // copy yarnrc config to generated SDK
       mkdirSync(sdkPackagePath, {recursive: true});
-      cpSync(path.join(sdkFolderPath, '.yarnrc.yml'), path.join(sdkPackagePath, '.yarnrc.yml'));
-      cpSync(path.join(sdkFolderPath, '.yarn'), path.join(sdkPackagePath, '.yarn'), {recursive: true});
+      cpSync(path.posix.join(sdkFolderPath, '.yarnrc.yml'), path.join(sdkPackagePath, '.yarnrc.yml'));
```

Comment:
why `posix`? `cpSync` is not supposed to be OS agnostic?

Reply:
I will remove it - I got pretty desesperate trying to figure out my missing yarn folder issue